src/app/legacy/containers/Include/amp/VjAmp.jsx

```diff
@@ -65,7 +64,7 @@ const StyledAmpIframe = styled(AmpIframe)`
     button {
       ${({ service }) => getSansBold(service)}
```

Comment:
Is there a rule here to when something should have a semi-colon at the end? For example, if I search `${({ service }) => getSansBold(service)}` in the code base I can see sometimes it has a` ; `at the end, and sometimes it doesn't.

Reply:
I think it probably should have the `;` at the end because it's a css property?

Reply:
Shayne and I talked about this and worked it out!

Reply:
getSansBold(service) returns styles with semi-colons already added. So I guess adding another semi-colon after this is optional. Semi-colons in css denote the end of a statement, so I guess something like: 
background-color: green;;;;
would be processed as one statement of: 'background-color: green' and then 3 empty statements.

Reply:
Shayne and I looked into this and it appears the semi-colon is redundent. In this instance, the function returns a block of CSS complete with semi-colons. There seems to be some logic at play that removes unnecessary semi-colons.

Here is a console log:

Without colon 
   font-family: ReithSans, Helvetica, Arial, sans-serif;
   font-weight: 700;
   font-style: normal;
  
With colon 
   font-family: ReithSans, Helvetica, Arial, sans-serif;
   font-weight: 700;
   font-style: normal;